conf/application.conf

```diff
@@ -202,6 +202,7 @@ urls {
   appealLink = "https://www.gov.uk/guidance/pension-administrators-register-a-scheme#appeal"
   applyToRegisterLink = "https://www.gov.uk/guidance/pension-administrators-register-a-scheme"
   pensionsRegulatorLink = "http://www.thepensionsregulator.gov.uk/trustees/authorisation-new-master-trusts.aspx"
+  featureToggle = "/admin/feature-toggles/%s"
```

Comment:
As this is a scheme toggle would it not be better to use pensions-scheme backend to store the toggle?